pkg/operator/ceph/csi/csi.go

```diff
@@ -268,6 +268,7 @@ func (r *ReconcileCSI) setParams(ver *version.Info) error {
 	csiRBDPodLabels := k8sutil.GetValue(r.opConfig.Parameters, "ROOK_CSI_RBD_POD_LABELS", "")
 	CSIParam.CSIRBDPodLabels = k8sutil.ParseStringToLabels(csiRBDPodLabels)
 	CSIParam.CSIClusterName = k8sutil.GetValue(r.opConfig.Parameters, "CSI_CLUSTER_NAME", "")
+	CSIParam.ImagePullPolicy = k8sutil.GetValue(r.opConfig.Parameters, "ROOK_CSI_IMAGE_PULL_POLICY", DefaultCSIImagePullPolicy)
```

Comment:
check `validateCSIVersion` function. we create a job from it you need to use csiImagePullPolicy for that job also.

Reply:
> check `validateCSIVersion` function. we create a job from it you need to use csiImagePullPolicy for that job also.

Done!